tests/ui/conftest.py

```diff
@@ -3,9 +3,19 @@
 import urlparse
 
 from fxapom.fxapom import DEV_URL, PROD_URL, FxATestAccount
+from pytest_django import live_server_helper
 import jwt
 import pytest
 import requests
+import json
+
+from django.core.management import call_command
+from olympia.amo.tests import create_switch
+
+
+@pytest.fixture(scope='session')
+def base_url(base_url):
+    return base_url or os.getenv('PYTEST_BASE_URL')
```

Comment:
The latest version of pytest-base-url already checks `PYTEST_BASE_URL`. Are you using the latest version?